src/main/java/frc/robot/subsystems/StateMachine.java

```diff
@@ -137,6 +133,13 @@ public Command tryState(RobotState desiredState) {
                 subLED);
           case CLIMBER_DEPLOYING:
           case CLIMBER_RETRACTING:
+            if (subClimber.climberPreped) {
+              Elastic.selectTab("Teleoperated");
+              return new None(subStateMachine, subCoralOuttake, subHopper, subAlgaeIntake, subClimber, subElevator,
+                  subLED);
+            } else {
+              Commands.print("Climber is not prepped!!! Not safe to return to NONE >____<");
+            }
```

Comment:
contents of the if statement need be swapped. also lets update the print message